commit bb387f15dd19c5c354e81c2ee1c62697c0a998c5
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-05-31T15:33:39-04:00 (4 months ago)
    
    TRITON-1710 update assets base image to triton-origin-x86_64-18.4.0
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 268f8db..43a1fc3 100644
--- a/Makefile
+++ b/Makefile
@@ -29,10 +29,10 @@ RELEASE_TARBALL=assets-pkg-$(ASSETS_PUBLISH_VERSION).tar.gz
 #
 # Stuff used for buildimage
 #
-BASE_IMAGE_UUID		= 04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f
+BASE_IMAGE_UUID		= a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME		= assets
 BUILDIMAGE_DESC		= SDC Assets
-BUILDIMAGE_PKGSRC	= nginx-1.11.1nb1
+BUILDIMAGE_PKGSRC	= nginx
 
 AGENTS = amon
 
diff --git a/deps/eng b/deps/eng
index 126c0f0..d25b8fc 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit 126c0f032b8bfddd45b14e8ee14e73e9798a013f
+Subproject commit d25b8fc60fb8c649559924870fe3aaf75e7421d5
